ios/engine/KMEI/KeymanEngine/Classes/Errors/SentryManager.swift

```diff
@@ -89,6 +91,107 @@ public class SentryManager {
     #endif
   }
 
+  private static func mapLoggingLevel(_ level: Sentry.SentryLevel) -> XCGLogger.Level {
+    switch(level) {
+        case .none:
+          return XCGLogger.Level.none
+        case .debug:
+          return .debug
+        case .info:
+          return .info
+        case .warning:
+          return .warning
+        case .error:
+          return .error
+        case .fatal:
+          return .severe
+        default:
+          return .info
+      }
+  }
+
+  /**
+   * Captures a Sentry event and copies its message to the engine's logging mechanism.
+   * If the logging level is not specified, the Sentry event's log-level will be used as a default.
+   *
+   * Will safely bypass the Sentry component if not activated by the app, only logging the
+   * message in such scenarios.
+   */
+  public static func captureAndLog(_ event: Sentry.Event, logLevel: XCGLogger.Level? = nil) {
+    // Guarded in case a library consumer decides against initializing Sentry.
+    if _started {
+      SentrySDK.capture(event: event)
+    }
+
+    let level = logLevel ?? mapLoggingLevel(event.level)
```

Comment:
You've can handle event.info, but I see you've replaced all the 
`log.info` with `breadcrumbAndLog`.

Reply:
There are a few spots with `log.debug` for breadcrumbs, and there's at least one case coming with a `.fatal` sentry-level error instead of plain `.error`.